package.json

```
@@ -6,7 +6,8 @@
   "dependencies": {
     "chalk": "4.1.2",
     "dayjs": "1.10.7",
-    "dayjs-random": "^1.0.0"
+    "dayjs-random": "^1.0.0",
```

```suggestion
    "dayjs-random": "1.0.0",
```
Версии должны быть фиксированными